Fixed; multiple-rpc-port-error-bug; (#2041)
substrate-developer-hub/substrate-docs: content/md/en/docs/tutorials/build-a-blockchain/simulate-network.md

   --chain local \
   --alice \
   --port 30333 \

>    --rpc-port 9945 && \
>    --rpc-port 9933 \

Reviewer: ```suggestion
   --rpc-port 9945 \
```

| `--chain local`                                             | Specifies the chain specification to use. Valid predefined chain specifications include `local`, `development`, and `staging`.                                                                                                             |
| `--alice`                                                   | Adds the predefined keys for the `alice` account to the node's keystore. With this setting, the `alice` account is used for block production and finalization.                                                                             |
| `--port 30333`                                              | Specifies the port to listen on for peer-to-peer (`p2p`) traffic. Because this tutorial uses two nodes running on the same physical computer to simulate a network, you must explicitly specify a different port for at least one account. |

> | `--rpc-port 9945`                                            | Specifies the port to listen on for incoming WebSocket and RPC traffic. The default port is `9944`. This tutorial uses a custom web socket port number (`9945`).                                                                                   |

Reviewer 2: ```suggestion
| `--rpc-port 9945`                                            | Specifies the port on which the server will listen for incoming JSON-RPC traffic via WebSocket and HTTP. The default port is `9944`. This tutorial uses a custom web socket port number (`9945`).                                                                                   |
```
For reference: https://github.com/paritytech/polkadot/issues/7363

   --bob \
   --port 30334 \
   --rpc-port 9946 \
   --telemetry-url "wss://telemetry.polkadot.io/submit/ 0" \
   --validator \
   --bootnodes /ip4/127.0.0.1/tcp/30333/p2p/12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp
   ```

   Note the following differences between this command and the previous one:

>    - Because the two nodes are running on the same physical computer, you must specify different values for the `--base-path`, `--port`, and `--rpc-port` options.

Reviewer 2: ```suggestion
   - Because the two nodes are running on the same physical computer, you must specify different values for the `--base-path`, `--port` and `--rpc-port` options.
```

Reviewer 2: sorry for the nit-pick

   --chain local \
   --bob \
   --port 30334 \

>    --rpc-port 9946 && \
>    --rpc-port 9934 \

Reviewer: ```suggestion
   --rpc-port 9946 \
```

Author: Thank you for the input @ghe0. I knew the nodes could start with only rpc port 9945, but I thought the problem was deeper than that, as far configuring the CLI itself and the runtime. Consensus is we move on without rpc ports 9933 && 9934.

Reviewer: Yes I think this is fine.

However we need to ensure we update the rest of the docs on this page to match. Please check to make sure we don't mention these removed ports anywhere else, and also mention that *both* RPC and WS is exposed on `--rpc-port`.
